rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2021-02-11
+nightly-2021-11-02
```

Comment:
nit: missing trailing newline

Reply:
Fixed, added an editorconfig too with a rule for final newline